libs/pulumi-lambda/src/lib/deny-log-group-create.ts

```diff
@@ -0,0 +1,29 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as aws from '@pulumi/aws'
+
+export class DenyLogGroupCreationPolicy extends pulumi.ComponentResource {
+    readonly policy: aws.iam.Policy
+
+    constructor(
+        name: string,
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        _: {},
+        opts?: pulumi.ComponentResourceOptions | undefined,
+    ) {
+        super('wanews:deny-log-group-create-policy', name, {}, opts)
+
+        this.policy = new aws.iam.Policy('deny-log-group-creation', {
```

Comment:
I'm still not completely across the rules on when to use it and when not to, but is this missing:
`{ parent: this }`

Reply:
Good catch, should always have `parent: this` inside a ComponentResource